### DISCUSSION TITLE
Logo Design Contest Winner

### DISCUSSION THREAD

**Discussion by Tyree** [immich-app/immich#7705] [General]
@alextran1502 Since comments were locked on the logo contest thread, I just sent you a message on Discord (mattaton1975).

Thanks so much to you and the team for choosing my design! A most welcome surprise today! :-) 

I love Immich, so I am stoked to work with you guys in some way!

**Comment by Eternal85** [comment]
what font did you use?

**Reply by Tyree** [reply]
> what font did you use?

I think it was Router.
https://fonts.adobe.com/fonts/router
